src/engines/witcher/lua/bindings.h

```diff
@@ -424,6 +424,667 @@ class LuaBindings {
 	static int luaBor(lua_State *state);
 };
 
+struct CAuroraSettingsData {
+	uint32_t _staticTimeFlow;
```

Comment:
Can you use `uint32` instead of `uint32_t`, to be consistent with the rest of the codebase?

(Yes, doing that was a mistake on my part, back when I started xoreos, but now we're kinda stuck with it.)